content/docs/communities/moderation/_index.md

```diff
@@ -0,0 +1,114 @@
++++
+title = "Community Moderation"
+weight = 100
+[extra]
+emoji = "🛡️"
+tile = "I want to deploy moderation tools"
+last_updated = "2022-10-27T08:00:00Z"
++++
+
+## Power levels
+
+Matrix has a basic system of roles, sometimes also called Power Levels. It is a
+number usually going from 0 to 100. Advanced users can decide to use a different
+range, but we will not cover it here for the sake of simplicity.
+
+By default, there are three roles in Matrix:
+
+| Power Level | Role          |
+|-------------|---------------|
+| 0           | User          |
+| 50          | Moderator     |
+| 100         | Administrator |
+
+By default:
+
+- **Users** can send messages, media, reactions and redact their own messages in
+  a room.
```

Comment:
Is redacting your own messages something that can be configured away? This makes it sounds like that a bit.

Reply:
It's not, but I wanted to give an overview of the user abilities. Do you have better phrasing in mind?

Reply:
Could be a separate statement righr before or after the listing that everybody can redact their own messages?